frontend/packages/client/src/components/Community/JoinCommunityButton.js

```diff
@@ -112,7 +112,7 @@ export default function JoinCommunityButton({
         onClick={isMember ? leaveCommunity : joinCommunity}
       >
         <Svg name="Eye" />
-        <Svg name="HideEye" />
+        <Svg name="EyeHidden" />
```

Comment:
changed the convention to `Icon<State>` so files are easier to read. This matches `BellHidden` and `BellFilled`